themes/soe_basic/src/scss/theme/_teasers.scss

```diff
@@ -3,6 +3,7 @@
 .ptype-stanford-entity {
   .su-entity-item {
     a {
+      text-decoration: none;
 
```

Comment:
@imonroe is the [soe_basic repo](https://github.com/SU-SOE/soe_basic) actively being reference? Should this be moved into the repo instead?

Ignoring my PR oopsie, it looks like the last active work was in 2022 
<img width="1369" height="532" alt="image" src="https://github.com/user-attachments/assets/11840ba6-53cd-4e31-a4ed-eefce97a3c59" />